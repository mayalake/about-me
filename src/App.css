.App {
  font-family: palatino;
}

.Header{
	text-align: center;
	color: white;
	background: linear-gradient(rgba(0,0,0,.3), rgba(0,0,0,.3)), url(./img/background.jpg);
	background-position: center;
	background-repeat: no-repeat;
	background-size: cover;
	height: 100vh;
}

.HeaderText{
	position: absolute;
	top: 50%;
	left: 50%;
	transform: translate(-50%, -50%);
}

.Footer{
	text-align: center;
	color: grey;
	font-size: 12px;
}
