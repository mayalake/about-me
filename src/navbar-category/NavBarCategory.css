.NavBarCategory {
	margin: 0 20px;
	background-color: transparent;
	border: none;
	color: white;
	font-size: 20px;
	text-decoration: none;
}

.NavBarCategory:hover {
	text-decoration: underline;
	cursor: default;
}