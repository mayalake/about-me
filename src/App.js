import React from 'react';

import NavBar from './navbar/NavBar';
import AboutMeSection from './about-section/AboutMeSection.jsx';
import ResumeSection from './resume-section/ResumeSection';
import ContactSection from './contact-section/ContactSection';

import './App.css';

const App = () =>
    <div className="App">
    	<header className="Header">
			<NavBar />
			<div className="HeaderText">
				<h1>Maya Lake</h1>
				<p>Software Developer<br/>Los Angeles</p>
			</div>
		</header>
		<AboutMeSection />
		<ResumeSection />
		<ContactSection />
		<footer className="Footer">&#169; 2020 Maya Lake</footer>
    </div>;

export default App;
