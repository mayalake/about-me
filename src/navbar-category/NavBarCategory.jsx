import React from 'react';
import PropTypes from 'prop-types';

import './NavBarCategory.css';

const NavBarCategory = (props) =>
<a className="NavBarCategory" href={props.link}>{props.name}</a>;

NavBarCategory.propTypes = {
	name: PropTypes.string.isRequired,
	link: PropTypes.string.isRequired
}

export default NavBarCategory;