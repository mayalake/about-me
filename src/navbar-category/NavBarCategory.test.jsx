import React from 'react';
import { render } from '@testing-library/react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import App from '../App';
import NavBarCategory from './NavBarCategory';

Enzyme.configure({ adapter: new Adapter() });

const navBarCategoryDetail = {name:"name", link:"link"};
let wrapped = Enzyme.shallow(<NavBarCategory name={navBarCategoryDetail.name} link={navBarCategoryDetail.link}/>);

describe('NavBarCategory', () => {
	it('renders the NavBarCategory correctly', () =>{
		expect(wrapped).toMatchSnapshot();
	});
});