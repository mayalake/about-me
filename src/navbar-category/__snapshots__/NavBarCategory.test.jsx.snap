// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBarCategory renders the NavBarCategory correctly 1`] = `ShallowWrapper {}`;
